week4/assignment/로그인 & 회원가입 👤/src/components/common/Input.jsx

```diff
@@ -0,0 +1,72 @@
+import { useState, useEffect } from "react";
+import { Sign } from "../../styles/commonStyle";
+
+import DoubleCheck from "../signup/DoubleCheck";
+
+import { PAGE } from "../../constants/page";
+import { INPUT } from "../../constants/input";
+import { CHECK } from "../../constants/check";
+
+import loginIcon from "../../assets/icons/Message.svg";
+import passwordIcon from "../..//assets/icons/Password.svg";
+import UserIcon from "../../assets/icons/User.svg";
+
+const Input = ({
+  pageType,
+  inputType,
+  onInputChange,
+  isAvailable,
+  setIsAvailable,
+}) => {
```

Comment:
사실 나도 이렇게 했는데 .. props로 받아와서 구조분해할당으로 주는게 조금 더 깔끔한거 같기도 해... 내 개인적인 생각이야..! 

```suggestion
const Input = (props) => {
const { pageType, inputType, onInputChange, isAvailable, setIsAvailable } = props;
```

Reply:
마쟈마쟈!!